app/code/community/Bolt/Boltpay/Helper/ApiTrait.php

```diff
@@ -99,6 +99,7 @@ private function verify_hook_api($payload, $hmacHeader)
             return $response == 200;
         } catch (Exception $e) {
             $this->notifyException($e);
+            $this->logErrorToDataDog($e);
```

Comment:
can we have generic name like `logError`? 

 - In the future we may switch logging backend from datadog to our own
 - In the future we may want to allow logging to local file system (log error to datadog, log warning to file system, etc)

Having generic naming allows us to change logging details w/o changing every place we call these method